### ISSUE TITLE
JAWS with Internet Explorer 11 doesn't read out number of results available

### ISSUE THREAD

**Issue by accessiblewebuk** [alphagov/accessible-autocomplete#373]
JAWS with Firefox does read out the number of results, but it doesn't work with Internet Explorer. Tested with JAWS 17 and JAWS 18 using IE11 on Windows 10.

**Comment by nickcolley** [comment]
There's improvements to how this part of the component works in 2.0.0 which has been tested in JAWs so I think this is resolved :)
